Description: Run bootloader hooks from /etc/initramfs/post-update.d after making
 the image, and add $NO_POST_UPDATE_HOOKS to disable this
 .
 See <https://kernel-team.pages.debian.net/kernel-handbook/ch-update-hooks.html#s-initramfs-hooks>
Author: наб <nabijaczleweli@nabijaczleweli.xyz>
Bug-Debian: https://bugs.debian.org/753752
Forwarded: not-needed
Last-Update: 2022-11-17

--- a/dracut.sh
+++ b/dracut.sh
@@ -2673,4 +2673,10 @@
     fi
 fi
 
+# Invoke policy-conformant bootloader hooks
+if [ -z "$NO_POST_UPDATE_HOOKS" ] && [ -d /etc/initramfs/post-update.d/ ]; then
+    run-parts --arg="${kernel}" --arg="${outfile}" \
+        /etc/initramfs/post-update.d/
+fi
+
 exit 0
